doc/source/whatsnew/v1.1.0.rst

```diff
@@ -288,6 +288,7 @@ Other enhancements
 - :meth:`HDFStore.put` now accepts `track_times` parameter. Parameter is passed to ``create_table`` method of ``PyTables`` (:issue:`32682`).
 - Make :class:`pandas.core.window.Rolling` and :class:`pandas.core.window.Expanding` iterable（:issue:`11704`)
 - Make ``option_context`` a :class:`contextlib.ContextDecorator`, which allows it to be used as a decorator over an entire function (:issue:`34253`).
+- ``Series.plot`` fix asymmetric error bars (:issue:`9536`)
```

Comment:
Could you described the previous and the current behavior to make the fix a bit more clear?

Reply:
I extended the description in https://github.com/pandas-dev/pandas/pull/34514/commits/4a2eb9cbb4753ed6b1fc2c9e14fe4ef3b4fc6f18